src/iop/tonecurve.c

```diff
@@ -344,6 +347,19 @@ void process(struct dt_iop_module_t *self, dt_dev_pixelpipe_iop_t *piece, const
                                          : d->table[ch_b][CLAMP((int)(b_in * 0x10000ul), 0, 0xffff)]);
         }
       }
+      else if(autoscale_ab == DT_S_SCALE_MANUAL_LCH)
+      {
+        const float pi = 3.14159f;
```

Comment:
pi is already set as a global constant `M_PI` I think

Reply:
ok. done